Sprint-1/exercises/decimal.js

```diff
@@ -1,4 +1,12 @@
 const num = 56.5678;
+let wholeNumberPart = Math.floor(num);
+console.log(wholeNumberPart);
+
+let decimalPart = num - Math.floor(num);
+console.log(decimalPart);
```

Comment:
If you run the program, you may notice that console.log() outputs the value of `decimalPart` with more than 4 decimal places (which is normal because of floating point arithmetic). 

If you were asked to print the value of `decimalPart` to exactly 4 decimal places, how would you modify your code?